sdk/spring/azure-spring-boot/src/test/java/com/azure/spring/autoconfigure/aad/AzureJwtBearerTokenAuthenticationConverterTest.java

```diff
@@ -0,0 +1,39 @@
+package com.azure.spring.autoconfigure.aad;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import java.time.Instant;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.Test;
+import org.springframework.security.authentication.AbstractAuthenticationToken;
+import org.springframework.security.oauth2.jwt.Jwt;
+
+public class AzureJwtBearerTokenAuthenticationConverterTest {
```

Comment:
Do we have tid?

Reply:
I'm going to add a method to the UserPrincipal to get the `tid`.